eng/pipelines/maintenance-packages-pr.yml

```diff
@@ -117,8 +117,9 @@ stages:
         - script: eng/common/cibuild.sh
             --configuration $(_BuildConfig)
             --prepareMachine
-            /p:SkipTests=true
+            /p:Test=false
             /p:IsPackable=true
+            /p:UnsetIsPackableForNonSourceProjects=true
             $(_AdditionalBuildArgs)
```

Comment:
I assume we should start using these new conditions when building locally. We should add it to the readme. You don't have to do it now, let's add a TODO comment to https://github.com/dotnet/maintenance-packages/pull/39

Reply:
You mean when you want to mimic this CI leg? Yes, for that `/p:IsPackable=true /p:UnsetIsPackableForNonSourceProjects=true` needs to be passed in. Where's the right place to document that?

Reply:
Yeah. I often times run the same command locally when I need to confirm the packages are built fine.

Feel free to either document it or add a TODO comment to https://github.com/dotnet/maintenance-packages/pull/39 , anywhere you think it would fit better.